### DISCUSSION TITLE
Time to retire "Master"?

### DISCUSSION THREAD

**Discussion by derwin12** [xLightsSequencer/xLights#4886] [General]
Joking/not joking... thoughts on changing make master, copy to master, master view etc ?

Thoughts of changing to "Render View" .. "Copy to Render View", etc.
Basically remove the word master everywhere?
Would be a big change both to the code and the docs.

**Comment by SopherConklin** [comment]
I think  Render View/Copy to Render View would make it super obvious exactly what is going on.  Plus, you'll never have to say "The Master View is used for Render order", again.  The experts already know what that does - they won't be confused by this - your audience with this text is new users, and this will make it much more intuitive for them.
